README.md

```diff
@@ -37,19 +33,11 @@ $ kubectl create secret generic kong-admin-token-secret --from-literal=kong-admi
 secret/kong-admin-token-secret created
 ```
 
-2. Set up collector, overriding Kong Admin host, servicePort and token to ensure
-   Kong Admin API is reachable by collector, this will allow collector to push
-   swagger specs to Kong.
-
-```console
-$ helm install my-release ./charts/kong-collectorapi --set kongAdmin.host=my-kong-kong-admin
-```
-
 ```console
```

Comment:
Should there be a quick blurb before this section to say what it's doing? Eg, "Check that that the token works:"

Reply:
I like this ^^